addons/hr_holidays/static/src/dashboard/time_off_card.js

```diff
@@ -56,24 +57,23 @@ export class TimeOffCard extends Component {
             (acc, data) => acc + data.amount,
             0
         );
+        this.accrualExcess = data.allows_negative
+            ? -data.exceeding_duration > data.max_allowed_negative
+            : -data.exceeding_duration > 0;
```

Comment:
This should be in a separate method to be called and recomputed so that it is also updated with `updateWarning`